src/main/java/org/alfresco/repo/domain/node/AbstractNodeDAOImpl.java

```diff
@@ -116,10 +116,10 @@
  */
 public abstract class AbstractNodeDAOImpl implements NodeDAO, BatchingDAO
 {
-    private static final String CACHE_REGION_ROOT_NODES = "N.RN";
-    private static final String CACHE_REGION_NODES = "N.N";
-    private static final String CACHE_REGION_ASPECTS = "N.A";
-    private static final String CACHE_REGION_PROPERTIES = "N.P";
+    public static final String CACHE_REGION_ROOT_NODES = "N.RN";
+    public static final String CACHE_REGION_NODES = "N.N";
+    public static final String CACHE_REGION_ASPECTS = "N.A";
+    public static final String CACHE_REGION_PROPERTIES = "N.P";
     

```

Comment:
Only `CACHE_REGION_NODES` is referenced in `DBQueryEngine` to create the node cache. Since the caches in the latter class are for temporary use I guess we should just change the visibility for that single constant and add a comment so that we remember to change it back when all temporary code is removed.

Reply:
Ok, I got it, let them all public and wait for other reviewers take.